package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gatsby-starter-hello-world",
```

Comment:
@CIPHERTron Don't forget to change name and description next time 😆

Reply:
@abhibhaw BIG OOPS XD